packages/data-stores/src/subscriber/types.ts

```diff
@@ -52,5 +51,3 @@ export type GetSubscribersImportsResponse = ImportJob[];
 export interface SubscriberDispatch {
```

Comment:
Just realized this type is not in use either.

Reply:
Good catch! Removed it in https://github.com/Automattic/wp-calypso/pull/74430/commits/2c4d57fe1b5b0746ab027ccb0697039ec67f5986